packages/cli/src/api/formats/csv.ts

```diff
@@ -0,0 +1,52 @@
+import fs from "fs"
+
+import { writeFileIfChanged } from "../utils"
+import { MessageType } from "../types"
+
+const serialize = (catalog) => {
+  var row = ""
+  for (const key of Object.keys(catalog)) {
+    var cur = catalog[key]
+    row += `\"${key}\"`
+    row += `,\"${cur.translation}\"` || ""
```

Comment:
What if the key or translation contain a separator (comma in this case)? Shouldn't we escape the value? Feel free to use any external dependency for CSV files. I'm planning to move formats to external packages anyway, so it wouldn't add dependency to everyones project.

Reply:
I have [enclosed the field in quotes](https://stackoverflow.com/questions/4617935/is-there-a-way-to-include-commas-in-csv-columns-without-breaking-the-formatting), so we can include separators in either keys or translations. I have tested with `,` and `"`. It works fine in my current project. I'd love to explore some of the external packages but couldn't find a 2 ways conversion library. Any suggestions?

Reply:
Hmm, that's really interesting. I would expect it to fail, if the message has one double quote, for example. 

AFAIK, if the message is unquoted, then comma is problem. If the message is surrounded by quotes, then the quotes inside the message become a problem and you need to escape them properly.

I'm not aware of any CSV packages, I'll check them later tonight.

Reply:
You are right, it does fail in the case that you mentioned. I will try to find some CSV packages too. Thank you for reviewing my PR.

Reply:
Hi @tricoder42 , I used the [papaparse](https://www.papaparse.com/) package to convert the csv file. Please take a look. Thank you.

Reply:
I have created tests for it as well. But still didn't pass the coverage check.